modules/tfm/tfm/common/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(TFM_PARTITION_PLATFORM)
+  set(src ${CMAKE_CURRENT_LIST_DIR}/src)
+  set(nrf_dir ${CMAKE_CURRENT_LIST_DIR}/../../../../)
```

Comment:
Isn't there some standard way to do this? :P

Reply:
I haven't found anything in the TFM build system which allows us to do thin without depending on the placement of the TFM module in relation to the nrf repo.